terraform/modules/tf_lambda/variables.tf

```diff
@@ -0,0 +1,132 @@
+// Note: We use this variable because terraform does not support "count" for module resources
+// https://github.com/hashicorp/terraform/issues/953
+variable "enabled" {
```

Comment:
Has there been an example where this would be used? I'm trying to contrast this approach with just deleting the module altogether

Reply:
Yes - the threat intel downloader is optional. Once the downloader uses this module (instead of building the Lambda resource directly), it needs a way to indicate whether or not the function should be enabled. This is, for example, how the [optional BinaryAlert CarbonBlack downloader](https://github.com/airbnb/binaryalert/blob/2f0ee271e266e220a278a1000162febc8c38d7ba/terraform/lambda.tf#L3) works

Reply:
I see, so your vision is to pre-allocate the Terraform code, and then just flip the switch as needed.  Generally with the SA CLI, we would just not marshal out the Terraform code to generate the Lambda.  Either approach works though.

Reply:
Ah, I see. Yeah, having the feature just adds flexibility - whichever way works best